.github/workflows/main.yml

```diff
@@ -1,6 +1,11 @@
 name: tests
 
-on: [push, pull_request]
+on:
+  merge_group:
+  push:
+    branches-ignore:
+    - gh-readonly-queue/**  # Temporary merge queue-related GH-made branches
```

Comment:
Where is gh-readonly-queue documented? Can we link to the authoritative documentation that recommends this change?